package.json

```diff
@@ -20,6 +20,7 @@
     "serverless": "~1.2.1"
   },
   "dependencies": {
+    "bluebird": "^3.4.7",
```

Comment:
Other libraries will bring this in to support older versions of node, but anything after 0.12 supports Promise natively.